src/main/java/com/coveo/configuration/parameterstore/strategy/ParameterStorePropertySourceEnvironmentPostProcessStrategy.java

```diff
@@ -0,0 +1,8 @@
+package com.coveo.configuration.parameterstore.strategy;
+
+import org.springframework.core.env.ConfigurableEnvironment;
+
+public interface ParameterStorePropertySourceEnvironmentPostProcessStrategy
+{
+    void postProcess(ConfigurableEnvironment environment);
```

Comment:
Following my rename comment  I'd rename this method to configureParameterStorePropertySources(...)